src/main/webapp/js/reducers/entity-reducer.js

```diff
@@ -31,6 +32,7 @@ const initialState = {
   [PERMISSION_ENTITY]: {},
   [CAMPAIGN_MESSAGE_ENTITY]: {},
   [USER_ENTITY]: {},
+  [APP_SETTINGS_ENTITY]: {},
```

Comment:
I don't this you should store it as an entity, it will only complicate things
create a separate reducer instead